.husky/eslint.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+STAGED_FILES=$(git diff --cached --name-only --diff-filter=d | grep '(\.js\|\.jsx$\|\.ts\|\.tsx)$')
+
+# Exit if no files. Passing no arguments would trigger eslint for whole repo
+if [ -z "$STAGED_FILES" ]; then
+  echo "No staged JavaScript/TypeScript files to lint."
+else
+  echo "Linting JavaScript/TypeScript files..."
+  echo "$STAGED_FILES"
+  echo ""
+
+  # No quotes to pass as separate arguments (files)
+  # shellcheck disable=SC2086
+  if ! yarn lint:js:fix-files $STAGED_FILES; then
+    echo "Eslint failed! Please fix the errors and try again."
```

Comment:
I can imagine someone wanting to commit some work in progress. The message could hint at `--no-verify` option which skips this check.